.env.example

```diff
@@ -1,4 +1,4 @@
-
 CONTENTFUL_TOKEN=test
 CONTENTFUL_SPACE=space
 MEETUP_KEY=test
+GATSBY_ENVIRONMENT="development"
```

Comment:
Quick way to share environment variables into our gatsby build (https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript)